pack/fhv/fhvc.json

```
@@ -118,9 +118,419 @@
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "flavor": "It was very dark, for the trees were grotesquely huge and luxuriant, and incredibly numerous.\n – H. P. Lovecraft and Winifred V. Jackson, \"The Green Meadow\"",
+    "illustrator": "Pixoloid Studios",
+    "name": "Tangled Thicket",
+    "pack_code": "fhvc",
+    "position": 94,
+    "quantity": 1,
+    "shroud": 2,
+    "text": "As an additional cost to move out of Tangled Thicket, you must spend an action.",
+    "traits": "Bog.",
+    "type_code": "location"
+  },
+  {
+    "code": "10594b", 
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "flavor": "The calm ripples hide something sinister deep below.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Open Water",
+    "pack_code": "fhvc",
+    "position": 94,
+    "quantity": 1,
+    "shroud": 5,
+    "traits": "Sunken.",
+    "text": "<b>Forced</b> – After you move into this location: Draw the top card of the encounter deck.",
+    "type_code": "location"
+  },
+  {
+    "back_link": "10595b",
+    "clues": 1,
+    "code": "10595a",
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The stagnant waters, lately quite devoid of animal life, now teemed with a horde of slimy enormous frogs which piped shrilly and incessantly...\n – H. P. Lovecraft, \"The Moon-Bog\"",
+    "illustrator": "Pixoloid Studios",
+    "name": "Fetid Pool",
+    "pack_code": "fhvc",
+    "position": 95,
+    "quantity": 2,
+    "shroud": 2,
+    "text": "This location gets +1 shroud for each sinkhole on it.",
+    "traits": "Bog.",
+    "type_code": "location"
+  },
+  {
+    "code": "10595b", 
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The calm ripples hide something sinister deep below.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Open Water",
+    "pack_code": "fhvc",
+    "position": 95,
+    "quantity": 2,
+    "shroud": 5,
+    "traits": "Sunken.",
+    "text": "<b>Forced</b> – After you move into this location: Draw the top card of the encounter deck.",
+    "type_code": "location"
+  },
+  {
+    "back_link": "10596b",
+    "clues": 0,
+    "code": "10596a",
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 11,
+    "faction_code": "mythos",
+    "flavor": "Stepping between the sunken boards allows you to navigate the flooded pathway.",
+    "illustrator": "David Frasheski",
+    "name": "Flooded Path",
+    "pack_code": "fhvc",
+    "position": 96,
+    "quantity": 2,
+    "shroud": 1,
+    "text": "[fast] During your turn, place 1 damage on this location, as a sinkhole: <b>Move</b> <i>(to a connecting location)</i>.",
+    "traits": "Bog.",
+    "type_code": "location"
+  },
+  {
+    "code": "10595b", 
```

This has the wrong code?

```suggestion
    "code": "10596b",
    "clues": null,
```